content/articles/getting-started-with-machine-learning-using-matlab/index.md

```diff
@@ -0,0 +1,99 @@
+### GETTING STARTED WITH MACHINE LEARNING IN MATLAB
+### Introduction
+Machine learning is a type of computer science that allows computer programs to learn and improve on their own. In the past, computers could only do what they are programmed to do but with machine learning, they could behave like human being and gain knowledge based on their past experiences. let's say for example we want a program that that can tell the difference between apples and bananas. First we give the computer some label pictures of each. The program will identify the pattern in the fruit and begin to remember them. It can then use these memories to identify the unlabeled and tell which is which.
+Machine learning is encountered in daily basis for example, in the medical industry, it is applied for predicting life span, organizing patients data e.t.c. Matlab provides a machine learning toolbox that makes machine learning easier. In this article, we will look at the basis of machine learning.
+
+### Prerequisite
+- [Matlab](https://www.mathworks.com/products/get-matlab.html?s_tid=gn_getml) installed.
+- [Data](https://archive.ics.uci.edu/ml/datasets/Bank+Marketing) for analysis
+- Proper understanding of [matlab](https://www.section.io/engineering-education/getting-started-with-matlab/) basics.
+
+### Challenges of machine learning
+- lots of data with many variables(predictors) and algorithms.
+- Data is too complex to know the governing equation.
+- Significant expertise is required.
+- No 'one size fits all' solution- this means it requires an iterative approach that is;  
+  - Try multiple algorithms, see what works best.
+- Time-consuming.
+
+### Why Matlab for machine learning
+Matlab does not only provides a strong environment for iterative exploration, but it also has apps that allow you to explore the impacts of different models on your data. The framework makes it easy to iterate over different algorithms and measure their accuracy.
+One can also incorporate a parallel computing paradigm to speed up computations. Once you are happy with the models, You can integrate them into larger data and analytics and decision making workflow.
+
+### Overview 0f machine learning
+Machine learning is classified into two branches, that is;
+- Supervised learning.
+- Unsupervised learning.
+
+### Unsupervised learning
+Here, you group your data based on some similarities or characteristics. It is not a must you know the grouping. This grouping can be done using clustering techniques. Clustering is the grouping of the dataset based on its characteristics. Matlab provides a variety of clustering algorithms. The algorithm is later grouped into partitioning algorithms and overlapping algorithms.
+
```

Comment:
The distinction between partition and overlapping algorithms is not clear. More explanation is required.